plugins/svgtools/include/detray/plugins/svgtools/conversion/landmark.hpp

```diff
@@ -0,0 +1,38 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s)
+#include "detray/geometry/surface.hpp"
+#include "detray/intersection/intersection.hpp"
+#include "detray/plugins/svgtools/conversion/point.hpp"
+#include "detray/plugins/svgtools/meta/proto/landmark.hpp"
+
+namespace detray::svgtools::conversion {
+
+/// @returns The proto landmark of a detray intersection.
+template <typename point3_t, typename detector_t>
+inline auto landmark(
+    const typename detector_t::geometry_context& context,
+    const detector_t& detector,
+    const detray::intersection2D<typename detector_t::surface_type,
+                                 typename detector_t::transform3>&
+        d_intersection) {
+    using p_landmark_t = svgtools::meta::proto::landmark<point3_t>;
+
+    const typename detector_t::point3 dir{};
```

Comment:
```suggestion
    const typename detector_t::vector3 dir{};
```
This will btw not work with line surfaces, where we will need the actual direction of the ray, but that is for later